.travis/matrix-script.sh

```diff
@@ -31,7 +31,9 @@ if [ "${MATRIX_TYPE}" = "python" ]; then
 
 elif [ "${MATRIX_TYPE}" = "python-sharded" ]; then
 
-    SHARDED_TEST_APPS="form_processor sql_db couchforms case phone receiverwrapper"
+    SHARDED_TEST_APPS=form_processor
+    # commented out due to bug that discarded all but first item
+    #SHARDED_TEST_APPS="form_processor sql_db couchforms case phone receiverwrapper"
```

Comment:
@snopoke: curious why you took code from this PR and put it in https://github.com/dimagi/commcare-hq/pull/11573, thereby creating a merge conflict here? Are you holding off on merging this PR for some reason?